assets/directory_asset_bundle.h

```diff
@@ -20,17 +20,20 @@ class DirectoryAssetBundle : public mojo::asset_bundle::AssetBundle {
   DirectoryAssetBundle(mojo::InterfaceRequest<AssetBundle> request,
                        std::string directory,
                        ftl::RefPtr<ftl::TaskRunner> task_runner);
+  ~DirectoryAssetBundle() override;
 
   // mojo::assert_bundle::AssetBundle implementation:
   void GetAsStream(
       const mojo::String& asset_name,
       const mojo::Callback<void(mojo::ScopedDataPipeConsumerHandle)>& callback)
       override;
 
+  bool GetAsBuffer(const std::string& asset_name, std::vector<uint8_t>* data);
+
  private:
-  ~DirectoryAssetBundle() override;
+  std::string GetPathForAsset(const std::string& asset_name);
 
-  mojo::StrongBinding<mojo::asset_bundle::AssetBundle> binding_;
+  mojo::Binding<mojo::asset_bundle::AssetBundle> binding_;
```

Comment:
I don't think `mojo::StrongBinding` and `mojo::Binding` are quite the same. In case of the strong binding, this objects lifecycle is tied to that of the binding. Now, it is not clear how this object will be collected. I think this introduces a leak over whatever was present earlier (if the binding was ever torn down, which I don't think it was).